app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddClaimBusinessPage.scala.html

```diff
@@ -38,6 +39,10 @@
 
 @html = {
     <p class="govuk-body"><strong>@messages(s"$key.p1")</strong></p>
+    @govukPanel(Panel(
+        title = Text("Success!"),
```

Comment:
NABD - pop this in the messages file.

Reply:
Urgh! That shouldn't even be in there. Thought I'd got rid of it. Let me fix.